cli/commands.go

```diff
@@ -45,6 +76,10 @@ func CheckTools(c *cli.Context) error {
 // CreateCluster creates a new single-node cluster container and initializes the cluster directory
 func CreateCluster(c *cli.Context) error {
 
+	if err := checkClusterName(c.String("name")); err != nil {
```

Comment:
I guess checking the provided cluster name here is enough, isn't it?
In all the other functions, an invalid hostname won't raise an error except for "cluster not found" :man_shrugging:

Reply:
IMHO, the error message in those cases won't be as useful to user. Extra check can help them to catch the potential typo. The advantage is minor though.  I will be happy to drop them if you'd prefer.  WDYT?

Reply:
O.K. I removed those checks and merged patch 2 and 3, updated the pull request.

Reply:
You could've left them in there from a functionality perspective. I was just in doubt about the necessity of it, since here we don't really care if the user gives us a valid unix hostname.